app/src/main/java/org/p2p/wallet/home/HomeModule.kt

```diff
@@ -66,6 +66,7 @@ object HomeModule : InjectionModule {
         factoryOf(::UserTokensPolling)
         /* Cached data exists, therefore creating singleton */
         // todo: do something with this dependenices!
+        // todo: to eliminate all this hell, we could just migrate to hilt
```

Comment:
or just refactor logic in this call to small ones and split dependencies
migrating to a new stack is not always the solution